blinky-statik/src/main/java/org/spideruci/analysis/statik/AnalysisConfig.java

```diff
@@ -16,6 +16,8 @@
   public final static String ENTRY_METHOD = "entrymethod";
   public final static String DEBUG = "debug";
   public final static String CALL_GRAPH_ALGO = "callgraph-algo";
+  public final static String CLASSPATH = "classpath";
+  private final static String SUT_JARPATH = "sutjarpath";
```

Comment:
I made SUT_JARPATH private because some projects like our CallGraphSubject doesn't have any jar file needs to be included. It will be only accessible inside this package so that if user specifies jar path(s) then, it will be included as class path.